flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliTableauResultView.java

```diff
@@ -114,6 +108,30 @@ public void displayResults() throws SqlExecutionException {
         }
     }
 
+    private void printTerminatedFooter(AtomicInteger receivedRowCount) {
```

Comment:
Could you check the behaviour whether hive/presto prints the time cose when users cancel the execution.

Reply:
Thanks for the hint. Checked Trino that the time consumption will be printed after users cancelled https://github.com/trinodb/trino/blob/ae7849e5b81b0c0eff7e980222502bf392a9a7f3/client/trino-cli/src/main/java/io/trino/cli/StatusPrinter.java#L121
https://github.com/trinodb/trino/blob/ae7849e5b81b0c0eff7e980222502bf392a9a7f3/client/trino-cli/src/main/java/io/trino/cli/StatusPrinter.java#L350